requirements.txt

```diff
@@ -1,13 +1,15 @@
 environs==9.5.0 #all
 discord_webhook==1.3.0 #all
 requests==2.28.1 #all
+ratelimit #all
+rank-torrent-name #all
```

Comment:
you may want to set this to `rank-torrent-name==0.2.23`

as the upcoming update will break changes.  The next one will improve parsing speed, cover a lot more stuff, and improved ranking as well.